src/features/quick-search/QuickSearch.js

```diff
@@ -13,6 +13,7 @@ class QuickSearch extends Component {
     static propTypes = {
         children: PropTypes.node,
         className: PropTypes.string,
+        dividerIndex: PropTypes.number,
```

Comment:
If you have bandwidth, should convert this to flow types since we don't use proptypes anymore

Reply:
I'll do this ^

Reply:
done